test/integration_tests/prototype/ctc_decoder_integration_test.py

```diff
@@ -0,0 +1,27 @@
+from parameterized import parameterized
+from torchaudio_unittest.common_utils import skipIfNoCtcDecoder
+from torchaudio_unittest.common_utils.ctc_decoder_utils import is_ctc_decoder_available
```

Comment:
I am avoiding to reuse the unit test utilities in integration tests for the sake of decoupling.
Can we simply let the tests fail if CTC decode feature is not found?

Also, please use `@pytest.mark.parametrize` for parameterizing in integration. We established the use of `parameterized` in unit tests because the unit tests take OOP approach, and we could not use `pytest` in fbcode in early days.